libipl/p10/ipl0.C

```diff
@@ -485,7 +498,8 @@ static int ipl_set_ref_clock(void)
 		ipl_log(IPL_ERROR, "Invalid number (%d) of clock target found\n",
 							clock_count);
 
-		ipl_error_callback(IPL_ERR_CLK_FAILED);
+		// no error callback, since this should be a software
```

Comment:
Why you are removing this? We will lose the error log in the platform (BMC), right?

I think we discussed this in the other commit (previous PR).

Reply:
Ok.